docs/connectors/index.mdx

```diff
@@ -6,51 +7,92 @@ on a growing library of **[input](sources)** and **[output](sinks)** connectors.
 
 ## Basics
 
-A connector is a Feldera API object that describes an external data source or sink such as a database
-table or a Kafka topic. Connectors work together with other API objects, namely programs and pipelines,
-to build end-to-end streaming analytics pipelines.
-
-* A [program](/docs#programs) defines a set of SQL tables and views.
-* A [pipeline](/docs#pipelines) consists of a program with input and output connectors attached to it.
-* A connector must explicitly be attached to a table (for input)
-  or view (for output) of a pipeline.
-* A connector can be attached to the tables/views of one or more pipelines,
-  and can even be attached to multiple tables (for input) or views (for output)
-  within the same pipeline.
-* Connectors are managed through the API under the **/v0/connectors/...** endpoints.
-  They can be retrieved
-  [individually](https://www.feldera.com/api/fetch-a-connector-by-name) or
-  [as a list](https://www.feldera.com/api/fetch-connectors-optionally-filtered-by-name-or-id),
-  [created](https://www.feldera.com/api/create-a-new-connector),
-  [overwritten](https://www.feldera.com/api/create-or-replace-a-connector),
-  [modified](https://www.feldera.com/api/update-the-name-description-and-or-configuration-of-a-connector) and
-  [deleted](https://www.feldera.com/api/delete-an-existing-connector).
-* If changes are made to the connectors of a running pipeline, they will only be
-  applied when the pipeline is restarted.
+A connector is a JSON object that describes an external data source or sink such as a database
+table or a Kafka topic. A SQL table can have multiple source connectors attached to it, specified as a list of JSON connector objects
+under the `'connectors'` attribute in the `WITH` clause of the table.
+
+Here is an example, where the `VENDOR` table has one connector, configured to fetch and insert JSON data
+from an HTTP URL, and a `VENDOR_VIEW` which sends the changes of the view
+to a Kafka topic in a format that can be consumed by Debezium:
+
+```sql
+create table VENDOR (
```

Comment:
We seem to use unusual and inconsistent SQL-styles throughout our examples, docs and posts. Let's stick to the common way of having identifiers in lower case (vendor, not VENDOR), and SQL syntax in upper case. So this would be:

```
CREATE TABLE vendor (
  id BIGINT NOT NULL PRIMARY KEY,
  name VARCHAR,
  some_other_field VARCHAR,
...
) WITH () ...
```

Reply:
I think the "more common" way is to use uppercase -- but anyways we seem to be using _mostly_ uppercase so far mostly in the docs, we should probably change it all at once during a refactoring

Reply:
this is a religious war that you better not get into
maybe we should build a sqlfmt tool to end the disputes

Reply:
Whatever we do, pick one and stick with it, and preferably, use a formatter. Right now, we use a mix of styles even in the same program and even in the same query!

Reply:
My suggestion for a style guide around casing at least: https://docs.telemetry.mozilla.org/concepts/sql_style